manim/mobject/shape_matchers.py

```diff
@@ -107,6 +107,27 @@ def __init__(self, mobject, stroke_color=RED, stroke_width=6, **kwargs):
 
 
 class Underline(Line):
+    """Creates an underline.
+
+    Parameters
+    ----------
+    Line
+        The underline.
+
+    Examples
+    --------
+    .. manim:: UnderLine
+
+        class UnderLine(Scene):
+            def construct(self):
+                m = Tex("M")  # Starting letter
+                man = Tex("M", "anim")  # Full Word
+                ul = Underline(man)  # Underlining the word
+                self.play(Write(m), run_time=1.5)
+                self.play(ReplacementTransform(m, man[0]), FadeIn(man[1]))  # Replaces the starting letter to word
```

Comment:
In general, I think the example is fine but a bit too simple to be a video; if you *insist* on making it a video it could be made a super short 1 second clip where the setup for the word Manim is done in advanced and the act of underlining is the *only* thing animated e.g.:

```suggestion
                man = Tex("M", "anim")  # Full Word
                self.add(man)
                ul = Underline(man)  # Underlining the word
```

Or better yet, it could just be made an image using `:save_last_frame:`.

Reply:
+1 for only :save_last_frame:

Reply:
+1